### ISSUE TITLE
Donate UAV Data

### ISSUE THREAD

**Issue by vesnikos** [OpenDroneMap/ODMdata#12]
Hi,

I got some UAV data from my Master's course that I could donate for the project,

~215 images with corresponding image locations and 44 target locations measured with RTK

![swindale](https://cloud.githubusercontent.com/assets/6684178/26551703/b8b30428-447b-11e7-8a14-99f3e2d2e584.jpg)

**Comment by coreysnipes** [comment]
@vesnikos I realize it's been a while... is this dataset still available for the ODMData sample data catalog?

**Comment by vesnikos** [comment]
Yes, it was part of my Msc project that I finished some time ago.

 I will need to take it out from deep storage and dig out all the parameters but I don't see any problems as long  my supervisor and I are acknowledged :)

**Comment by coreysnipes** [comment]
@vesnikos yes, definitely happy to acknowledge you and your supervisor on this.  Thanks for your help.

**Comment by vesnikos** [comment]
Hi @coreysnipes, if you could head over at https://github.com/vesnikos/Swindale-UAV-Images (branch revision, should be the default) and check that it has all the technical details you guys need. 

Also, do you guys have a licence that you want me to release the data under?

**Comment by coreysnipes** [comment]
I'll check the specs.  @smathermather @pierotofy do you have a preference on license? 
Thanks!

**Comment by pierotofy** [comment]
Creative Commons or Public Domain (choice is yours).

**Comment by smathermather** [comment]
I would say avoid Public Domain, as (for what I understand, I am not a lawyer), public domain cannot be declared in Francophone legal systems, but must be conferred by law otherwise.  CC0 should be used instead of public domain for full international application (unless you are acting as a public agency which can put things into public domain for all legal systems).

**Comment by pierotofy** [comment]
Ah, interesting. Did not know.
